.github/workflows/ci-mount-inspect.yaml

```diff
@@ -0,0 +1,83 @@
+# Run weekly on main, Sundays at 23:00 UTC (6 pm CDT)
+on:
+  pull_request:
+  #schedule:
+  #  - cron: '0 23 * * 0'
+  #push:
+  #  branches:
+  #    - main
```

Comment:
Right now I'm running on PRs to test, but the idea would be to run this periodically on the images in our container. Thinking maybe once a week? But I'm open to other ideas/suggestions for how we may want this to look.